packages/documentation/postcss.config.js

```diff
@@ -0,0 +1,7 @@
+const postcssPseudoClasses = require('./postcss-pseudo-classes.js');
+
+module.exports = () => {
+  return {
+    plugins: [postcssPseudoClasses],
```

Comment:
I wasn't able to change the Vite config to use this plugin like https://github.com/vitejs/vite/discussions/8693